domain/db_test.go

```diff
@@ -56,3 +60,61 @@ func TestDomainSession(t *testing.T) {
 	_, err = se.Execute(context.Background(), "create global binding for select * from t where i>100 using select * from t use index(index_t) where i>100")
 	require.NoError(t, err)
 }
+
+func TestNormalSessionPool(t *testing.T) {
+	lease := 50 * time.Millisecond
```

Comment:
Increasing the lease maybe could prevent some weird failures.

Reply:
done